Watching for file changes with StatReloader
Method Not Allowed: /handle_request/
[02/Jan/2025 18:50:57] "GET /handle_request/ HTTP/1.1" 405 52
Method Not Allowed: /handle_request/
[02/Jan/2025 18:50:59] "GET /handle_request/ HTTP/1.1" 405 52
[02/Jan/2025 18:51:01] "GET /dashboard/ HTTP/1.1" 200 23123
Method Not Allowed: /handle_request/
[02/Jan/2025 18:51:05] "GET /handle_request/ HTTP/1.1" 405 52
[02/Jan/2025 18:51:15] "GET /admin_hr_login/ HTTP/1.1" 200 1771
[02/Jan/2025 18:51:15] "GET /static/css/static/images/background.png HTTP/1.1" 404 2027
[02/Jan/2025 18:51:19] "GET / HTTP/1.1" 200 3280
[02/Jan/2025 18:51:21] "POST / HTTP/1.1" 302 0
[02/Jan/2025 18:51:21] "GET /user_login/ HTTP/1.1" 200 5793
[02/Jan/2025 18:51:34] "POST /user_login/ HTTP/1.1" 302 0
[02/Jan/2025 18:51:34] "GET /exam_instructionv/ HTTP/1.1" 200 2224
[02/Jan/2025 18:51:35] "GET /automatic_selection/? HTTP/1.1" 200 3293
[02/Jan/2025 18:51:38] "GET /exam_guidelines/? HTTP/1.1" 200 3155
[02/Jan/2025 18:51:38] "GET /static/css/exam_guidelines.css HTTP/1.1" 304 0
[02/Jan/2025 18:51:40] "POST /start_test/ HTTP/1.1" 302 0
[02/Jan/2025 18:51:40] "GET /payment_start/ HTTP/1.1" 200 2665
[02/Jan/2025 18:51:44] "POST /initiate-payment/ HTTP/1.1" 200 4169
[02/Jan/2025 18:52:05] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:52:05] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:52:05] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:52:09] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:52:14] "POST /handle_request/ HTTP/1.1" 200 3361
[02/Jan/2025 18:52:14] "GET /static/css/payment_success.css HTTP/1.1" 200 4885
[02/Jan/2025 18:52:14] "GET /static/images/congrats-modal.png HTTP/1.1" 200 78778
[02/Jan/2025 18:52:20] "POST /start_test/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:20] "GET /quiz/6/question/94/ HTTP/1.1" 200 10397
[02/Jan/2025 18:52:20] "GET /static/css/question-1.css HTTP/1.1" 304 0
[02/Jan/2025 18:52:20] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:23] "POST /quiz/6/save_answer/94/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:23] "GET /quiz/6/question/95/ HTTP/1.1" 200 10400
[02/Jan/2025 18:52:23] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:25] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:25] "POST /quiz/6/save_answer/95/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:25] "GET /quiz/6/question/96/ HTTP/1.1" 200 10184
[02/Jan/2025 18:52:25] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:27] "POST /quiz/6/save_answer/96/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:27] "GET /quiz/6/question/97/ HTTP/1.1" 200 10389
[02/Jan/2025 18:52:27] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:29] "POST /quiz/6/save_answer/97/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:29] "GET /quiz/6/question/98/ HTTP/1.1" 200 10356
[02/Jan/2025 18:52:30] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:31] "GET /get_remaining_time/ HTTP/1.1" 200 24
[02/Jan/2025 18:52:31] "POST /quiz/6/save_answer/98/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:31] "GET /quiz_summary/6 HTTP/1.1" 200 4857
[02/Jan/2025 18:52:31] "GET /static/css/start-test.css HTTP/1.1" 304 0
[02/Jan/2025 18:52:34] "POST /start_test/ HTTP/1.1" 302 0
[02/Jan/2025 18:52:34] "GET /finish_test/ HTTP/1.1" 200 1436
[02/Jan/2025 18:52:34] "GET /static/css/complete-congrats.css HTTP/1.1" 404 2006
[02/Jan/2025 18:53:09] "GET / HTTP/1.1" 200 3280
[02/Jan/2025 18:54:21] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:56:50] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 18:59:08] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:01:24] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:03:31] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:05:51] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:14:16] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:16:44] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:22:34] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:25:17] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:27:39] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:30:03] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 19:42:27] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:01:20] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:09:20] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:21:10] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:22:15] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:27:25] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:29:48] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 20:42:43] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 21:02:44] "GET / HTTP/1.1" 200 3280
[02/Jan/2025 21:03:50] "GET / HTTP/1.1" 200 3280
Not Found: /api/session/properties
[02/Jan/2025 21:03:50] "GET /api/session/properties HTTP/1.1" 404 14513
[02/Jan/2025 21:04:14] "GET /admin/ HTTP/1.1" 302 0
Not Found: /owncloud/status.php
[02/Jan/2025 21:04:15] "GET /owncloud/status.php HTTP/1.1" 404 14504
Not Found: /status.php
[02/Jan/2025 21:04:36] "GET /status.php HTTP/1.1" 404 14477
[02/Jan/2025 21:04:53] code 400, message Bad request version ('À\x14À')
[02/Jan/2025 21:04:53] You're accessing the development server over HTTPS, but it only supports HTTP.
Not Found: /console
[02/Jan/2025 21:05:21] "GET /console HTTP/1.1" 404 14468
Not Found: /ext-js/app/common/zld_product_spec.js
[02/Jan/2025 21:05:34] "GET /ext-js/app/common/zld_product_spec.js HTTP/1.1" 404 14558
[02/Jan/2025 22:30:56,472] - Broken pipe from ('87.251.64.11', 63553)
[02/Jan/2025 22:39:14] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 22:55:06] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 22:57:02] "POST /handle_request/ HTTP/1.1" 200 68
[02/Jan/2025 22:58:53] "POST /handle_request/ HTTP/1.1" 200 68
/home/ubuntu/crtd_new/payment_integration/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
January 02, 2025 - 18:50:50
Django version 5.1.4, using settings 'CRTD.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

Request Method: GET
Request Method: GET
Request Method: GET
Fullname:CRTD Technologes, email:crtdgroup@gmail.com, contact:8815165433
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {'razorpay_order_id': ['order_PegEa8f1MUSSUM'], 'razorpay_payment_id': ['pay_PegEjOLRpa8OGD'], 'razorpay_signature': ['e61ef4314684e08d95b247bd558ee4c036cc3691657a80ac522e15146c210b5b']}>
Order ID: order_PegEa8f1MUSSUM
Signature: e61ef4314684e08d95b247bd558ee4c036cc3691657a80ac522e15146c210b5b
payment_id: pay_PegEjOLRpa8OGD
Found Payment object: Payment order_PegEa8f1MUSSUM - pending
Order order_PegEa8f1MUSSUM current status: pending
Signature verification result: True
User email: crtdgroup@gmail.com
User paid status: True
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Request Method: POST
Request Data: <QueryDict: {}>
Order ID: None
Signature: None
payment_id: None
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1073, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1010, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/management/commands/runserver.py", line 134, in inner_run
    self.check(display_num_errors=True)
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/management/base.py", line 486, in check
    all_issues = checks.run_checks(
                 ^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/checks/urls.py", line 16, in check_url_config
    return check_resolver(resolver)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/checks/urls.py", line 26, in check_resolver
    return check_method()
           ^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/urls/resolvers.py", line 531, in check
    for pattern in self.url_patterns:
                   ^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/urls/resolvers.py", line 718, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/urls/resolvers.py", line 711, in urlconf_module
    return import_module(self.urlconf_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/crtd_new/CRTD/urls.py", line 26, in <module>
    path('', include('payment_integration.urls', namespace='payment_integration')),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/urls/conf.py", line 39, in include
    urlconf_module = import_module(urlconf_module)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/crtd_new/payment_integration/urls.py", line 2, in <module>
    from .views import initiate_payment, handle_request, payment_start
  File "/home/ubuntu/crtd_new/payment_integration/views.py", line 24
    <<<<<<< HEAD
    ^^
SyntaxError: invalid syntax
/home/ubuntu/crtd_new/payment_integration/views.py changed, reloading.
Performing system checks...

Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1073, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1010, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/management/commands/runserver.py", line 137, in inner_run
    self.check_migrations()
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/core/management/base.py", line 581, in check_migrations
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/db/migrations/loader.py", line 58, in __init__
    self.build_graph()
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/db/migrations/loader.py", line 229, in build_graph
    self.load_disk()
  File "/home/ubuntu/crtd_new/venv/lib/python3.12/site-packages/django/db/migrations/loader.py", line 120, in load_disk
    migration_module = import_module(migration_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/ubuntu/crtd_new/quiz/migrations/0006_alter_question_question_text.py", line 1
    <<<<<<< HEAD
    ^^
SyntaxError: invalid syntax
/home/ubuntu/crtd_new/quiz/migrations/0006_alter_question_question_text.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
Watching for file changes with StatReloader
